cli/src/commands/obslog.rs

```diff
@@ -63,17 +65,24 @@ pub(crate) fn cmd_obslog(
     let workspace_command = command.workspace_helper(ui)?;
 
     let start_commit = workspace_command.resolve_single_rev(&args.revision)?;
-    let wc_commit_id = workspace_command.get_wc_commit_id();
 
     let diff_formats =
         diff_util::diff_formats_for_log(command.settings(), &args.diff_format, args.patch)?;
+    let with_content_format = LogContentFormat::new(ui, command.settings())?;
 
     let template_string = match &args.template {
         Some(value) => value.to_string(),
         None => command.settings().config().get_string("templates.log")?,
     };
-    let template = workspace_command.parse_commit_template(&template_string)?;
-    let with_content_format = LogContentFormat::new(ui, command.settings())?;
+
+    let template;
+    let commit_node_template;
+    {
+        let language = workspace_command.commit_template_language()?;
+        template = workspace_command.parse_template(&language, &template_string)?;
```

Comment:
nit: ditto. `template_string` can also be moved in this block.

Reply:
Done.